src/BuildScriptGeneratorCli/Commands/BuildScriptCommand.cs

```diff
@@ -61,7 +67,18 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 return ProcessConstants.ExitFailure;
             }
 
-            console.WriteLine(generatedScript);
+            if (string.IsNullOrEmpty(OutputPath))
```

Comment:
Get the full path of this string since users can supply relative (`./foo.sh` or `../foo.sh`) or full paths